.devcontainer/socket_proxy.py

```
@@ -0,0 +1,91 @@
+    listen_path = sys.argv[1]
+    target_path = sys.argv[2]
+
+    # Write PID to stdout so parent shell can capture it and kill it later
+    print(os.getpid(), flush=True)
+
+    server = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+    if os.path.exists(listen_path):
+        try:
+            os.remove(listen_path)
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)